lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -763,6 +763,15 @@ static void optimizePool(Function *F) {
   } // For all nodes in the graph.
 }
 
+static Variable *getModifiableVariable(Node &node) {
```

Comment:
Doxygen comments please.

Also, the name is not great, because "Modifiable" is subjective and depends on the context.